tests/phpunit/tests/rest-api/rest-global-styles-revisions-controller.php

```diff
@@ -0,0 +1,236 @@
+<?php
+/**
+ * Unit tests covering WP_REST_Global_Styles_Revisions_Controller functionality.
+ *
+ * @package WordPress
+ * @subpackage REST API
+ *
+ * @covers WP_REST_Global_Styles_Revisions_Controller
+ *
+ * @group restapi-global-styles
+ * @group restapi
+ */
+class WP_REST_Global_Styles_Revisions_Controller_Test extends WP_Test_REST_Controller_Testcase {
+	/**
+	 * @var int
+	 */
+	protected static $admin_id;
+
+	/**
+	 * @var int
+	 */
+	protected static $second_admin_id;
+
+	/**
+	 * @var int
+	 */
+	protected static $author_id;
+
+	/**
+	 * @var int
+	 */
+	protected static $global_styles_id;
+
+	public function set_up() {
+		parent::set_up();
+		switch_theme( 'tt1-blocks' );
+	}
+
+	/**
+	 * Create fake data before our tests run.
+	 *
+	 * @param WP_UnitTest_Factory $factory Helper that lets us create fake data.
+	 */
+	public static function wpSetupBeforeClass( $factory ) {
+		self::$admin_id        = $factory->user->create(
+			array(
+				'role' => 'administrator',
+			)
+		);
+		self::$second_admin_id = $factory->user->create(
+			array(
+				'role' => 'administrator',
+			)
+		);
+		self::$author_id       = $factory->user->create(
+			array(
+				'role' => 'author',
+			)
+		);
+		// This creates the global styles for the current theme.
+		self::$global_styles_id = $factory->post->create(
+			array(
+				'post_content' => '{"version": ' . WP_Theme_JSON::LATEST_SCHEMA . ', "isGlobalStylesUserThemeJSON": true }',
+				'post_status'  => 'publish',
+				'post_title'   => __( 'Custom Styles', 'default' ),
+				'post_type'    => 'wp_global_styles',
+				'post_name'    => 'wp-global-styles-tt1-blocks',
+				'tax_input'    => array(
+					'wp_theme' => 'tt1-blocks',
+				),
+			),
+			true
+		);
+	}
+
+	/**
+	 * @covers WP_REST_Global_Styles_Controller::register_routes
+	 */
+	public function test_register_routes() {
+		$routes = rest_get_server()->get_routes();
```

Comment:
This is failing on CI (tests pass locally for me for some reason)

`TypeError: array_keys(): Argument #1 ($array) must be of type array, bool given in /var/www/tests/phpunit/includes/factory/class-wp-unittest-factory-for-thing.php:171
`

Any hints? Did I miss an import/registration somewhere 🤔